tracing-error/src/lib.rs

```diff
@@ -148,7 +148,7 @@
 //!
 //! fn main() {
 //!     let subscriber = tracing_subscriber::Registry::default()
-//!         // any number of other subscriber layers may be added before or
+//!         // any number of other collector subscribers may be added before or
```

Comment:
@davidbarsky things like these are probably still lingering in places. Do you think it would be a good idea to keep this PR open for a while longer and we can fix things up as we see them and merge it all at once before releasing?

Reply:
If you notice leftover comments like these, we can include the fixes for them in this PR. However, I'd prefer to merge this PR in sooner rather than later to avoid this PR from rotting—let's set a deadline of merging this in by tomorrow or so?

Reply:
I'm fine merging now too, as you prefer.